units/systemd-logind.service.in

```diff
@@ -22,6 +22,12 @@ After=dbus.socket
 [Service]
 BusName=org.freedesktop.login1
 CapabilityBoundingSet=CAP_SYS_ADMIN CAP_MAC_ADMIN CAP_AUDIT_CONTROL CAP_CHOWN CAP_DAC_READ_SEARCH CAP_DAC_OVERRIDE CAP_FOWNER CAP_SYS_TTY_CONFIG CAP_LINUX_IMMUTABLE
+DeviceAllow=char-/dev/console rw
+DeviceAllow=char-alsa rw
```

Comment:
logind never opens audio devices, it only chmod()s/chown()s them. No need to allow write rw access here.

Reply:
or changes ACLs on them, which too should not require r or w access...

Reply:
ok